airflow/settings.py

```diff
@@ -543,6 +543,9 @@ def initialize():
 # loaded from module.
 LAZY_LOAD_PROVIDERS = conf.getboolean('core', 'lazy_discover_providers', fallback=True)
 
+# By default running listeners on scheduler is disabled. Set it to True if you want to execute them.
+EXECUTE_LISTENERS_ON_SCHEDULER = conf.getboolean('core', 'execute_listeners_on_scheduler', fallback=False)
```

Comment:
It needs to be added to https://github.com/apache/airflow/blob/main/airflow/config_templates/default_airflow.cfg and https://github.com/apache/airflow/blob/main/airflow/config_templates/config.yml too as we have got automated docs building from there - https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html

Reply:
Done. Also, added basic docs.